rollup.config.js

```diff
@@ -69,52 +62,53 @@ if (buildTarget !== buildTargets.DEV) {
+        format: "iife"
+      }
+    ],
+    plugins: minify ? [...plugins, baseCodeTerser] : plugins
+  });
+}
+
+config.push({
+  input: "src/standAlone.js",
   output: [
     {
       file: `${destDirectory}alloy${minifiedExtension}.js`,
-      // For the Reactor-specific build, we need to use the CommonJS format
-      // for output instead of IIFE, otherwise Rollup doesn't know whether the
-      // "external" modules (as defined elsewhere in this configuration) are
-      // coming from global variables or from another CommonJS bundler, so
-      // it adds a bunch of cruft to accommodate and logs build warnings.
-      // Because we have to use the CommonJS format for the Reactor-specific
-      // build, we can't just use an "intro" that looks like:
-      // if (IE less than version 11) {
-      //   console.warn("unsupported browser");
-      //   return;
-      // }
-      // because you can't "return" from a CommonJS module. Therefore, we have
-      // to do an if-else for an "intro" and "outro".
-      // If we do an if-else, we're not compliant with strict mode, because
-      // babel inserts its function declarations directly inside our if-else.
-      // Because function declarations are not allowed directly within an
-      // if-else in strict mode, we need to wrap babel's
-      // function declarations (and our own code) inside an IIFE.
-      // So, what we end up with is output using "format: cjs" but with an
-      // appropriate intro and outro that provides browser checking and an
-      // IIFE. This works for both Reactor-specific and standalone use cases.
-      format: "cjs",
-      // Allow non-IE browsers and IE11
-      // document.documentMode was added in IE8, and is specific to IE.
-      // IE7 and lower are not ES5 compatible so will get a parse error loading
-      // the library.
+      format: "iife",
       intro:
         "if (document.documentMode && document.documentMode < 11) {\n" +
         "  console.warn('The Adobe Experience Cloud Web SDK does not support IE 10 and below.');\n" +
-        "} else {\n" +
-        "  (function() {",
-      outro: "  })();\n}"
+        "  return;\n" +
+        "}\n"
     }
   ],
-  plugins
-};
+  plugins: minify ? [...plugins, terser()] : plugins
+});
 
-// If we're building for Reactor, we'll use Reactor's core modules
-// (named @adobe/reactor-*) instead of including the packages directly.
-if (buildTarget === buildTargets.PROD_REACTOR) {
-  config.external = name => /^@adobe\/reactor-/.test(name);
+if (buildTarget === buildTargets.PROD && !minify) {
+  config.push({
+    input: "src/index.js",
+    output: [
+      {
+        file: `${destDirectory}index.js`,
+        format: "es"
+      }
+    ],
+    // The @adobe/reactor-* dependencies are specified as peerDependencies so no need to include them in the
+    // module build. The Launch extension does not need them included.
+    external(name) {
+      return /^@adobe\/reactor-/.test(name);
+    },
```

Comment:
Another thing that we may have discussed but is worth describing at this point is that (1) when we build the Alloy extension, we're not running any sort of bundler on the extension library modules and (2) Launch's bundler that generates the Launch library doesn't support extension library modules requiring npm packages. In other words, our extension library module can't do `require("alloy")` because Launch doesn't do any npm package resolution except for Launch core modules (e.g., `require("@adobe/reactor-object-assign")`). Launch may support npm package resolution in the future, but that's a deeper discussion. We _could_ run our own bundler on our extension library code before we deliver our extension package to Launch, which would prevent Launch from having to do any npm resolution. 

But, with that said, our code in the extension library modules can require the alloy package by _path_ instead of by _name_, like this:

```
require("../../node_modules/@adobe/alloy/dist/index.js")'
```

Since Launch won't have to do any npm resolution, Launch just treats it as though it were any other relatively referenced file in the extension. This also means we don't have to run our own bundler on our extension library code either. Let me know if that doesn't make sense.

Reply:
Interesting. When I was working on the extension I had to add a rollup step to generate the alloy code, but it would be nice not to have to do that.